internal/core/core.go

```diff
@@ -51,6 +51,35 @@ type Axis struct {
 	Unit string `json:"unit" example:"ms"`
 } // @name Axis
 
+// @Description Geometrical plane with depth/time datapoints
+type RegularSurface struct {
+	// Horizon / height-map
+	Horizon [][]float32 `json:"horizon" binding:"required"`
```

Comment:
_if_ we break the API, maybe we want to use that opertunity to rename this field too? As it's inside a structure now, maybe `values` or `data` is better? xtgeo uses `values`